src/main/java/gov/cabinetoffice/gap/applybackend/service/SpotlightService.java

```diff
@@ -0,0 +1,58 @@
+package gov.cabinetoffice.gap.applybackend.service;
+
+import com.amazonaws.services.sqs.AmazonSQS;
+import com.amazonaws.services.sqs.model.MessageAttributeValue;
+import com.amazonaws.services.sqs.model.SendMessageRequest;
+import gov.cabinetoffice.gap.applybackend.config.SpotlightQueueConfigProperties;
+import gov.cabinetoffice.gap.applybackend.model.GrantMandatoryQuestions;
+import gov.cabinetoffice.gap.applybackend.model.GrantScheme;
+import gov.cabinetoffice.gap.applybackend.model.SpotlightSubmission;
+import gov.cabinetoffice.gap.applybackend.repository.SpotlightSubmissionRepository;
+import lombok.RequiredArgsConstructor;
+import lombok.extern.slf4j.Slf4j;
+import org.springframework.stereotype.Service;
+
+import java.time.Clock;
+import java.time.Instant;
+import java.util.UUID;
+
+@Slf4j
+@RequiredArgsConstructor
+@Service
+public class SpotlightService {
+
+    public static final String SPOTLIGHT_SUBMISSION_ID = "spotlightSubmissionId";
+    private final SpotlightSubmissionRepository spotlightSubmissionRepository;
+    private final AmazonSQS amazonSqs;
+    private final SpotlightQueueConfigProperties spotlightQueueProperties;
+    private final Clock clock;
+
+    public void createSpotlightCheck(GrantMandatoryQuestions mandatoryQuestions, GrantScheme scheme) {
+
+        // save a spotlight submission object to the database
+        final SpotlightSubmission spotlightSubmission = SpotlightSubmission.builder()
+                .mandatoryQuestions(mandatoryQuestions)
+                .grantScheme(scheme)
+                .version(scheme.getVersion()) //TODO is this how we want to set the version?
```

Comment:
I think we should decouple the spotlight submission version from the scheme version. There is only one version right now, so I think this should be a static value of `1`

Reply:
Awesome, will make that change now